src/libs/ReportUtils.ts

```diff
@@ -9299,6 +9301,39 @@ function getAllReportActionsErrorsAndReportActionThatRequiresAttention(
         reportAction = getReportActionWithSmartscanError(reportActionsArray);
     }
 
+    if (!isReportArchived && report?.statusNum === CONST.REPORT.STATUS_NUM.OPEN) {
+        // Find the most recent DEW_SUBMIT_FAILED action
+        const mostRecentDewSubmitFailedAction = reportActionsArray
+            .filter((action) => isDynamicExternalWorkflowSubmitFailedAction(action))
+            .reduce<ReportAction | undefined>((latest, current) => {
+                if (!latest || (current.created && latest.created && current.created > latest.created)) {
+                    return current;
+                }
+                return latest;
+            }, undefined);
+
+        if (mostRecentDewSubmitFailedAction) {
+            // find the most recent SUBMITTED action
+            const mostRecentSubmittedAction = reportActionsArray
+                .filter((action) => isSubmittedAction(action))
+                .reduce<ReportAction | undefined>((latest, current) => {
+                    if (!latest || (current.created && latest.created && current.created > latest.created)) {
+                        return current;
+                    }
+                    return latest;
+                }, undefined);
+
+            const shouldShowDEWError = !mostRecentSubmittedAction || (mostRecentSubmittedAction.created && mostRecentDewSubmitFailedAction.created > mostRecentSubmittedAction.created);
```

Comment:
`(mostRecentSubmittedAction.created && mostRecentDewSubmitFailedAction.created > mostRecentSubmittedAction.created);`

We don't need to check `mostRecentSubmittedAction.created` anymore.

`mostRecentDewSubmitFailedAction.created > mostRecentSubmittedAction.created;`